addons/fix-editor-comments/addon.json

```diff
@@ -38,30 +38,35 @@
     {
       "name": "Fix dragging blocks with comments",
       "id": "fix-drag",
+      "description": "Fixes a Scratch bug where if you move a block with a comment attached to it, the comment will go back to where it was before once you switch sprites or reload the project.",
       "type": "boolean",
       "default": true
     },
     {
-      "name": "Add scroll bars to clipped comments",
+      "name": "Add scroll bars",
       "id": "scroll",
+      "description": "Adds scroll bars to long comments and allows you to scroll through them with the scroll wheel.",
```

Comment:
From Transifex:

<img width="414" height="107" alt="Image" src="https://github.com/user-attachments/assets/f72ca514-a21b-41fc-8af8-8425cac9cea4" />

Reply:
> I would add "... scroll wheel or the touchpad"

I don't know...

But here's another wording:
```suggestion
      "description": "Allows you to scroll through long comments.",
```
This adds information onto the name "Add scroll bars" instead of also repeating the setting name, and does not assume any particular input device.

Reply:
Could the setting be called something like "Scroll through long comments" or "Make long comments scrollable"? Then a description wouldn't be needed.